sphinx/source/changelog.rst

```diff
@@ -3,6 +3,29 @@
 
 Change Log
 ==========
+Version 0.0.2 (April 29, 2022)
+--------------------------------
+
+Added the mounting object detection algorithm to detect and classify the mounting
+configuration of solar installations in satellite imagery. Updated several of the Github
+workflows to provide more rigorous testing protocols (requirements.txt check and flake8 check).
+
+Documentation
+~~~~~~~~~~~~~
+- Updated Sphinx Documentation to account for new functions.
+- Updated the Jupyter notebooks to reflect pipeline changes: adding in the mounting configuration detection classifier and running it on satellite imagery.
+
+Scripts
+~~~~~~~~~~~~~
+- Add the function :py:func:`panel_segmentation.panel_train.trainMountingConfigClassifier`.
+- Add the functions :py:func:`panel_segmentation.panel_detection.runSiteAnalysisPipeline` and :py:func:`panel_segmentation.panel_detection.classifyMountingConfiguration`.
+- Add unit testing for the :py:func:`panel_segmentation.panel_train.trainMountingConfigClassifier`, :py:func:`panel_segmentation.panel_detection.runSiteAnalysisPipeline`, 
+  and :py:func:`panel_segmentation.panel_detection.classifyMountingConfiguration` functions.
```

Comment:
None of these link correctly, I assume because they're missing the corresponding class names?

Reply:
Yep--addressed in commit https://github.com/NREL/Panel-Segmentation/pull/15/commits/41ecea125c343fc3db41b057994ba949b468f77b